platforms/core-configuration/kotlin-dsl-provider-plugins/src/main/kotlin/org/gradle/kotlin/dsl/provider/plugins/precompiled/DefaultPrecompiledScriptPluginsSupport.kt

```diff
@@ -353,6 +356,7 @@ private
 fun configureKotlinCompilerArguments(
     objects: ObjectFactory,
     implicitImports: ImplicitImports,
+    snapshotter: FileCollectionSnapshotter,
```

Comment:
💭 These are now needed because of the separation between `FileCollectionSnapshotter` and `FileCollectionFingerprinter`.